### PR TITLE
Issues/0050 colour space conversion examples

### PR DESCRIPTION
**Pull request by simontWork**
Added Javascript examples for Conversion between extended-sRGB/extended-linear-sRGB and HLG, and a suggested Tone-mapping to be used when showing HLG on an sRGB display.

Two Questions:

- Should we add Javascript examples of the various transfer functions mentioned?
- Should we link to the ITU guide for creating colour space to CIE XYZ matrices?

I've spoken to Timo about creating similar examples for PQ.

### CODE REVIEW

File: hdr_html_canvas_element.md [hunk 1]
```diff
@@ -100,13 +100,45 @@ WebGPU's ``GPUSwapChainDescriptor`` can allow for specifying higher bit depth fo
...
+
+* in the case where `rec2100-hlg` are used, tone mapping is performed by the display device. This is akin to the
+  scenario where the `src` of an `img` element is an HLG image.
 
 * in the case where `extended-linear-srgb` or `extended-srgb` are used, tone mapping is performed by the web app, using display capabilities provided by the platform.
 
 ![Tone mapping scenarios](./tone-mapping-scenarios.png)
 
+#### Suggested Tone mapping for HDR content on sRGB displays
+Not these are only used to tonemap HDR images at the point of rendering for display when the display is known to be an sRGB display.  They are not used for conversion between colour spaces which is defined in section XXXX.
```
Comment (lrosenthol) [inline hdr_html_canvas_element.md:114]: Not -> Note?

File: hdr_html_canvas_element.md [hunk 2]
```diff
@@ -157,7 +189,7 @@ The component signals are mapped to red, green and blue tristimulus values accor
 
 #### rec2100-pq
 
-The component signals are mapped to red, green and blue tristimulus values according to the PQ system system specified in Rec. ITU-R BT.2100.
+The component signals are mapped to red, green and blue tristimulus values according to the Perceptual Quantizer (PQ) system system specified in Rec. ITU-R BT.2100.
```
Comment (lrosenthol) [inline hdr_html_canvas_element.md:192]: Since PQ is used earlier in the doc (including in a section header), this should appear earlier

**Review by lrosenthol — COMMENTED** [review]
Some minor editorials

**Review by svgeesus — COMMENTED** [review]
This looks good to me

**Pull request merged**